dts/bindings/espi/microchip,xec-espi.yaml

```diff
@@ -44,3 +44,8 @@ properties:
       type: int
       description: soc group irq for eSPI peripheral channel
       required: true
+
+    uart_select:
+      type: int
+      description: soc uart to which eSPI host logical uart is mapped
```

Comment:
What exactly does this mean?

Reply:
eSPI bus can tunneled different traffic from different ports: uart, parallel, acpi, keyboard. eSPI hosts will interact with these ports using legacy port addresses, however eSPI slave HW intercept this traffic and can redirect it to any physical soc port. 
This is very similar to board dts mapping option to select the uart port used for console  (zephyr,console = &uart1;  )

Reply:
We should probably use a reference like &uart1 instead of having to have a magic value here.  But we can see if we fix that up in the future.

Reply:
Removed the uart_select field from device tree and added a KConfig option. 
Tried to use DT_CHOSEN_Z_CONSOLE as KConfig for console subsystem without success. This can be corrected in the future